src/OmniSharp.Script/ScriptProjectSystem.cs

```diff
@@ -79,16 +86,31 @@ public class ScriptProjectSystem : IProjectSystem
         private readonly Dictionary<string, ProjectInfo> _projects;
         private readonly OmniSharpWorkspace _workspace;
         private readonly IOmniSharpEnvironment _env;
+        private readonly ILoggerFactory _loggerFactory;
         private readonly ILogger _logger;
+        private readonly IAssemblyLoader _assemblyLoader;
+        private static readonly Lazy<string> _targetFrameWork = new Lazy<string>(ResolveTargetFramework);
+
+        private static string ResolveTargetFramework()
+        {
+            return Assembly.GetEntryAssembly().GetCustomAttributes()
```

Comment:
No problem. I was just making sure that I did not change the way metadata references are being added when running on the full desktop framework.